src/main/java/edu/ucsb/cs156/courses/controllers/PersonalCoursesController.java

```diff
@@ -71,8 +85,10 @@ public PersonalCourses addCourse(
             @ApiParam("enrollCd") @RequestParam String enrollCd,
             @ApiParam("psID") @RequestParam long psId,
             @ApiParam("Quarter (in yyyyq)") @RequestParam String quarter) {
-        CurrentUser currentUser = getCurrentUser();
-        log.info("currentUser={}", currentUser);
+        //FIX (Replace with course search method)
+        if(!STUBfunc(enrollCd,quarter)){
+            throw new EnrollCdInvalidException(enrollCd,quarter);
+        }
```

Comment:
This is good! 

But you should also validate the psId.

Do a personalScheduleRepository.find(psId) and if it doesn't find it, you should ALSO reject that as an invalid request.  You could throw an EntityNotFoundException.